### ISSUE TITLE
Select text in console with keyboard

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6077]
There should be a way to select text in the RStudio console using just the keyboard, so you can copy it. Currently you must use the mouse.

You can get focus to the previous output via Help / Accessibility / Tab Key Always Moves Focus, then Shift+Tab from the console prompt, but any attempt to select text bounces back to the prompt.

A model to consider would be something along the lines of Caret Browsing mode in FireFox and Edge.

Note that screen-reader users CAN copy text from the console using screen-reader commands, but it's clunky and varies considerably between screen readers, so a built-in way to do it that is the same for keyboard users and screen-reader users would be ideal.
